.github/workflows/check.yml

```diff
@@ -122,3 +122,8 @@ jobs:
         run: |
           echo ${{ secrets.APP_API_KEY }} > .test-token
           make e2e-test
+      - uses: actions/upload-artifact@v3
+        with:
+          name: e2e-test-logs
```

Comment:
There are 2 parallel runs for different python versions. Is it a problem that we use the same artifact name for both?